src/util/log.d

```diff
@@ -399,7 +423,6 @@ version (Posix)
         {
             import core.stdc.signal : signal, SIG_ERR;
             import core.sys.posix.signal : SIGHUP;
-            import std.exception : enforce;
```

Comment:
If you keep this local import and repeat the import locally in the helper function?
Then it's easier to get rid of obsolete imports.

Reply:
done